dev/ci/ci-basic-overlay.sh

```diff
@@ -339,3 +339,8 @@ project jasmin "https://github.com/jasmin-lang/jasmin" "main"
 # Lean Importer
 ########################################################################
 project lean_importer "https://github.com/SkySkimmer/coq-lean-import" "master"
+
+########################################################################
+# SerAPI
+########################################################################
+project serapi "https://github.com/ejgallego/coq-serapi" "v8.17"
```

Comment:
Do we not have a master branch to follow?

Reply:
We should indeed rename v8.17 back to master

Reply:
@ejgallego Will you do this now or later?